src/scss/Footer.scss

```diff
@@ -0,0 +1,57 @@
+@import './App.scss';
```

Comment:
- [OPTIONAL ]  According to the project requirement, Styling must be EXACTLY the same as the one provided on [Zeplin](https://zeplin.io/). Please make these in order to stick to the original design:

- What if we leave one bar instead of two between the Comments, Remove button, and Edit? actually, you have two 😜 
- Let's add this missing vertical bar to separate the `progressPercent` class div with the `progressStatus` class div😜 

- Your Design:

![555](https://user-images.githubusercontent.com/79058364/172933184-611e5354-6274-439a-8d41-dc95bac1d5de.png)



- Figma design

![bookstore](https://user-images.githubusercontent.com/79058364/162426330-10521c8c-526f-4fca-9450-dc3154d41b55.png)